gulpfile.js

```diff
@@ -25,11 +26,15 @@ var gulpRename = require('gulp-rename');
 var gulpReplace = require('gulp-replace');
 var gulpJsonTransform = require('gulp-json-transform');
 var Promise = require('bluebird');
-var requirejs = require('requirejs');
 var Karma = require('karma');
 var yargs = require('yargs');
 var AWS = require('aws-sdk');
 var mime = require('mime');
+var rollup = require('rollup');
+var rollupPluginStripPragma = require('./Tools/rollup-plugin-strip-pragma');
```

Comment:
Since this is a dev-dependency, and it's now published on npm, we should be able to remove `Tools/` here, right?

Reply:
Yes, good catch.